packages/client/src/components/navigationDrawer/LinkList.tsx

```diff
@@ -1,38 +1,80 @@
 import * as React from "react";
-import { List, ListItem, ListItemIcon, ListItemText } from "@material-ui/core";
-import { Link } from "react-router-dom";
-import { Info, PermMedia } from "@material-ui/icons";
-import { OverridableComponent } from "@material-ui/core/OverridableComponent";
-import { SvgIconTypeMap } from "@material-ui/core/SvgIcon/SvgIcon";
-import { makeStyles } from "@material-ui/core/styles";
 import { ReactElement } from "react";
+import { ChevronRight, ExpandMore, Folder, Info, PermMedia } from "@material-ui/icons";
+import gql from "graphql-tag";
+import { TreeView } from "@material-ui/lab";
+import { useQuery } from "@apollo/client";
+import { TreeGetProjectList } from "./__generated__/TreeGetProjectList";
+import { TreeGetDatasetList } from "./__generated__/TreeGetDatasetList";
```

Comment:
Hey friend, wrong import

Reply:
See other files. It is normal to import from generated files. Specifically, https://github.com/wpilibsuite/Axon/blob/0371b0ac03e6ec72b58696120665eba5feed0cd3/packages/client/src/components/datasets/Datasets.tsx#L17